engine/gamesys/src/dmsdk/gamesys/resources/res_convex_shape.h

```diff
@@ -23,14 +23,14 @@ namespace dmPhysics
 
 namespace dmGameSystem
 {
-    struct ConvexShapeResource
+    struct ConvexShapeResourceBox2D
```

Comment:
Should we keep the shape resource as before? I don't know if this is a breaking change or not, but I'd like to avoid the "m_3D" and having separate objects.. Maybe it should be just in the resource type?

`dmPhysics::HCollisionShape m_Shape;`

Reply:
I think it makes sense to keep it as a resource. I don't think we want to bake it into a particular shape as those are more lightweight and less reusable. The custom shape feels more relevant to have as a standalone resource.